tests/integration/gcp.py

```diff
@@ -519,21 +514,30 @@ def create_vm(self):
             },
         }
 
-        operation = self._compute.instances().insert(project=self.project, zone=self.zone, body=config).execute()
+        if self.config['secureboot']:
+            shielded_instance_config = {
```

Comment:
I wouldn't use a temporary variable here but instead directly add the "shielded_instance_config" to the `config` variable like this:
```
config["shielded_instance_config"] = {
  "enable_secure_boot": True,
  ...
}
```